From: Richard B Winters <rik@mmogp.com>
Date: Sun, 3 May 2015 20:54:20 -0400
Subject: Fix pcsd.service destination in Makefile

 - The pcsd.service file is installed to /usr/lib/systemd/
   system by default, this requires changing on debian systems

Change-Id: If33c6f26eb50e218ea1b97ee8f2bc0f616e23784
Signed-off-by: Richard B Winters <rik@mmogp.com>
---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/Makefile
+++ b/Makefile
@@ -45,8 +45,8 @@
 	install -m 700 -d ${DESTDIR}/var/lib/pcsd
 	install -m 644 -D pcsd/pcsd.logrotate ${DESTDIR}/etc/logrotate.d/pcsd
 ifeq ($(IS_SYSTEMCTL),true)
-	install -d ${DESTDIR}/usr/lib/systemd/system/
-	install -m 644 pcsd/pcsd.service ${DESTDIR}/usr/lib/systemd/system/
+	install -d ${DESTDIR}/lib/systemd/system
+	install -m 644 pcsd/pcsd.service ${DESTDIR}/lib/systemd/system/
 else
 	install -m 755 -D pcsd/pcsd ${DESTDIR}/${initdir}/pcsd
 endif
